ansible/templates/nginx.conf.j2

```diff
@@ -18,12 +18,36 @@ server {
     include proxy_params;
     proxy_pass http://api;
   }
+  location ~* \.(css|js) {
+    expires max;
+    break;
+  }
+  gzip on;
+  gzip_vary on;
+  gzip_disable "MSIE [1-6]\.";
+  gzip_comp_level 6;
+  gzip_min_length 1100;
+  gzip_buffers 16 8k;
+  gzip_proxied any;
+  gzip_proxied expired no-cache no-store private auth;
+  gzip_types
+      text/plain
+      text/css
+      text/js
+      text/xml
+      text/javascript
+      application/javascript
+      application/x-javascript
+      application/json
+      application/xml
+      application/rss+xml
+      image/svg+xml;
 }
 
-{% if env == 'production' %}
+{% if env != 'development' %}
 server {
   listen 80 default_server;
   server_name _;
-  return 301 https://{{fqdn}}$request_uri;
+  return 301 http://{{fqdn}}$request_uri;
```

Comment:
Once I implement SSL this will be listening on both protocols and we'll want to use https explicitly.